server/graphql/v2/input/OrderCreateInput.ts

```diff
@@ -72,5 +72,9 @@ export const OrderCreateInput = new GraphQLInputObjectType({
       type: OrderContextInput,
       description: 'Some context about how this order was created',
     },
+    isBalanceTransfer: {
+      type: GraphQLBoolean,
+      description: 'Whether this is transferring the remaining balance from a project/event',
```

Comment:
Or from a collective to its host.

Reply:
Thanks for pointing it out. Corrected that. 👍🏽